app/javascript/components/StudioAssessmentCard/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+class StudioAssessmentCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.showParticipant = this.showParticipant.bind(this);
+  }
+
+  showParticipant() {
+    const pId = this.props.assessment.participant_id;
+    window.location.assign(`participants/${String(pId)}`);
+  }
+
+  render() {
+    const ass = this.props.assessment;
```

Comment:
more descriptive names is usually better. I'd probably prefer longer var names if possible. `assessment` isn't too long. `pictureScore`, `programScore`, etc...